readme.md

```diff
@@ -236,6 +215,11 @@ Default: `false`
 
 If `true`, runs `command` inside of a shell. Uses `/bin/sh` on UNIX and `cmd.exe` on Windows. A different shell can be specified as a string. The shell should understand the `-c` switch on UNIX or `/d /s /c` on Windows.
 
+We recommend against using this option since it is:
+- not cross-platform, encouraging shell-specific syntax.
+- slower, because of the additional shell interpretation.
+- unsafe, potentially allowing command injection.
```

Comment:
This should be added here too: https://github.com/sindresorhus/execa/blob/df50cda9f4b0bd07fc6a9e709e223427cd9c977b/index.d.ts#L68